src/widgetastic/browser.py

```diff
@@ -440,6 +440,11 @@ def set_attribute(self, attr, value, *args, **kwargs):
             "arguments[0].setAttribute(arguments[1], arguments[2]);",
             self.element(*args, **kwargs), attr, value)
 
+    def size(self, *args, **kwargs):
```

Comment:
hmm, i wonder if we should perhaps name this one `size_of` instead of `size` 
since just `size` feels a bit property-ish